HMCL/src/main/resources/assets/lang/I18N.properties

```diff
@@ -308,27 +308,28 @@ folder.screenshots=Screenshots
 game=Game
 game.crash.info=Game Status
 game.crash.reason=Crash Analyzer
-game.crash.reason.duplciated_mod=Duplicated mods: %s.
-game.crash.reason.file_changed=Game file corrupted.
-game.crash.reason.gl_operation_failure=Game crashed due to some mod, shader packs, texture packs.
+game.crash.reason.duplciated_mod=The current game cannot run because of duplicate mod: %s.\n%s\nEach Mod can only be installed one, please delete the duplicate mod and try again.
+game.crash.reason.file_changed=The current game cannot run because file verification failed.\nIf you modify the Minecraft.jar, you need roll back the changes,or re-download the game.
+game.crash.reason.gl_operation_failure=Game crashed due to some mod,shader packs, texture packs.\nPlease disable the mod/shader packs/ texture packs you are using and try again.
```

Comment:
1. some mod,[missing space] shader packs, 
2. disable the mod[s]
3. do not add spaces after the slashes.